horizon/responses.go

```diff
@@ -31,6 +31,16 @@ type Account struct {
 	Signers              []Signer          `json:"signers"`
 }
 
+func (a Account) GetNativeBalance() string {
```

Comment:
I don't think we should panic if there is no native balance. I think you should just return 0. This would mean the account doesn't exist